### ISSUE TITLE
default filesystem disk is asked from the wrong config file

### ISSUE THREAD

**Issue by Thommis** [Pod-Point/laravel-mail-export#7]
in `pod-point\laravel-mail-export\src\StorageOptions.php` the default filesystem disk is asked from `config('filesystem.default')` , but in Laravel 7/8 it should be `config('filesystems.default')` as the filename is `filesystems` instead of `filesystem`

@see (pod-point\laravel-mail-export\src\StorageOptions.php)
```/**
     * Build the default storage disk using the config if none is provided by the developer.
     *
     * @return string
     */
    private function defaultDisk(): string
    {
        return config('mail-export.disk') ?: config('filesystem.default');
    }
```
@Laravel-Repo
[https://github.com/laravel/laravel/blob/7.x/config/filesystems.php](https://github.com/laravel/laravel/blob/7.x/config/filesystems.php)

**Comment by clemblanco** [comment]
Thanks @Thommis I believe this might be a typo. We will look into this for sure.

**Comment by clemblanco** [comment]
Thank you for contributing @Thommis this has now been released part of `0.1` and `0.2`.
